packages/node_modules/@webex/internal-plugin-conversation/src/encryption-transforms.js

```diff
@@ -151,6 +151,13 @@ export const transforms = toArray('outbound', {
 
   prepareActivityKmsMessage(ctx, key, activity) {
     if (activity.kmsMessage) {
```

Comment:
issue: I undid the changes in this file and ran the unit tests and they still passed so something is not being covered here